addons/udes_stock/tests/common.py

```diff
@@ -488,6 +488,18 @@ def get_picking_names(cls, pickings):
         names = pickings.mapped("name")
         return ", ".join(map(str, names))
 
+    @classmethod
+    def update_move_lines(cls, move_lines, qty=None, user=None):
+        if user is None:
+            user = cls.env.user
+        # We pass the user in the env as well, just for consistency
+        cls.env.real_uid = user.id
```

Comment:
Think we can drop this now - as traking doesn't revert to real_uid. The sudo calls don't (i beileve) mask the real user anymore.